pkg/bgp/manager/metallb.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package speaker abstracts the BGP speaker controller from MetalLB. This
+// package provides BGP announcements based on K8s object event handling.
+package manager
+
+import (
+	"context"
+	"os"
+
+	bgpconfig "github.com/cilium/cilium/pkg/bgp/config"
+	bgpk8s "github.com/cilium/cilium/pkg/bgp/k8s"
+	bgplog "github.com/cilium/cilium/pkg/bgp/log"
+	"github.com/cilium/cilium/pkg/option"
+	metallballoc "go.universe.tf/metallb/pkg/allocator"
+	metallbctl "go.universe.tf/metallb/pkg/controller"
+	"go.universe.tf/metallb/pkg/k8s"
+	"go.universe.tf/metallb/pkg/k8s/types"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Import ordering.

(Spent a few mins looking for tools that can auto detect this. Unfortunately, no tool exists that can detect our order of preference and auto group them together. :disappointed:)

Reply:
Not a problem. Will fix